cpp/example_code/s3/tests/test_create_bucket.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright Amazon.com, Inc. or its affiliates.All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+#include <aws/core/Aws.h>
+#include <aws/s3/model/BucketLocationConstraint.h>
+#include <awsdoc/s3/s3_examples.h>
+
+int main()
+{
+    Aws::String bucket_name = "my-bucket";
```

Comment:
A suggestion is to use a random bucket name and a cleanup function to delete the bucket.
If the bucket name is a constant string and no cleanup function, the test will fail the next time you run it.
A useful tool to generate random string is class `Aws::Utils::UUID` defined in `aws-sdk-cpp/aws-cpp-sdk-core/include/aws/core/utils/UUID.h`
```c++
Aws::String uuid = UUID::RandomUUID();
```